yarn.lock

```diff
@@ -1872,8 +1872,8 @@
     react-transition-group "^4.4.5"
 
 "@mui/monorepo@https://github.com/mui/material-ui.git#master":
-  version "5.13.7"
-  resolved "https://github.com/mui/material-ui.git#a4afa9ff2442589deb73a4b08c299913983967bd"
+  version "5.14.2"
+  resolved "https://github.com/mui/material-ui.git#c3a3413159f9c13421898366cfac41dca49c2371"
```

Comment:
That's odd, I still see the old side nav in the Netlify deployment https://deploy-preview-9187--material-ui-x.netlify.app/x/react-data-grid/

Could this be a Netlify caching issue?

<img width="290" src="https://github.com/mui/mui-x/assets/13808724/7dfc2fe3-206d-43de-9880-d00745f1ed6a">

Reply:
Yes, I rerun with cleaning netlify cach. Seems to be fixed 👍